### ISSUE TITLE
AttributeError: 'FileUnicodeMap' object has no attribute 'add_code2cid'

### ISSUE THREAD

**Issue by speedplane** [euske/pdfminer#49]
I am getting an exception when I try to `process_page` on the following PDF:
https://www.docketalarm.com/cases/PTAB/IPR2014-00396/Inter_Partes_Review_of_U.S._Pat._7310111/docs/02-20-2014-POR-1773/Power_of_Attorney-5-Power_of_Attorney.pdf

The PDF is digitally signed and I bet that has something to do with it.  I don't understand digital signatures or this code well enough to debug it. If you can spot the issue quickly, that would be great.

Stack trace:

```
 File "..\libs\pdfminer\pdfinterp.py", line 757, in process_page
    self.render_contents(page.resources, page.contents, ctm=ctm)
  File "..\libs\pdfminer\pdfinterp.py", line 770, in render_contents
    self.execute(list_value(streams))
  File "..\libs\pdfminer\pdfinterp.py", line 795, in execute
    func(*args)
  File "..\libs\pdfminer\pdfinterp.py", line 733, in do_Do
    interpreter.render_contents(resources, [xobj], ctm=mult_matrix(matrix, self.ctm))
  File "..\libs\pdfminer\pdfinterp.py", line 768, in render_contents
    self.init_resources(resources)
  File "..\libs\pdfminer\pdfinterp.py", line 339, in init_resources
    self.fontmap[fontid] = self.rsrcmgr.get_font(objid, spec)
  File "..\libs\pdfminer\pdfinterp.py", line 193, in get_font
    font = self.get_font(None, subspec)
  File "..\libs\pdfminer\pdfinterp.py", line 184, in get_font
    font = PDFCIDFont(self, spec)
  File "..\libs\pdfminer\pdffont.py", line 637, in __init__
    CMapParser(self.unicode_map, StringIO(strm.get_data())).run()
  File "..\libs\pdfminer\cmapdb.py", line 292, in run
    self.nextobject()
  File "..\libs\pdfminer\psparser.py", line 584, in nextobject
    self.do_keyword(pos, token)
  File "..\libs\pdfminer\cmapdb.py", line 354, in do_keyword
    self.cmap.add_code2cid(x, cid+i)
AttributeError: 'FileUnicodeMap' object has no attribute 'add_code2cid'
```

**Comment by qayshp** [comment]
I was hitting this bug and would like to ask a favor, I hope that's ok.

To help prevent everyone from needing to run setup.py install from the source, it would be awesome if you could update the PyPI package (to have this fix and others).
The latest on PyPI claims to be from 2014-03-28 uploaded by user @euske , and other users do not have permission to update it. The fix you mention, 6b6fc264ffd58cd1c305e8c23990b85e846964e9, was committed 2014-04-16.

Not being familiar with PyPI, I wanted to see what would happen if I tried to update the package. Gladly, and as expected, it wouldn't let me.
> Registering pdfminer to https://pypi.python.org/pypi
> Server response (403): You are not allowed to store 'pdfminer' package information

**Comment by RobinsonWei** [comment]
Same here, it would be great if the author could upload the package. I got the same error with 20140328 packages. 
Although I directly reference to the git to get it installed and fixed, I would be great if we could have a bug free version.
